druid/src/command.rs

```diff
@@ -14,28 +14,38 @@
 
 //! Custom commands.
 
-use std::any::Any;
-use std::sync::{Arc, Mutex};
+use std::any::{self, Any};
+use std::{
+    marker::PhantomData,
+    sync::{Arc, Mutex},
+};
 
 use crate::{WidgetId, WindowId};
 
+/// A [`Selector`]s identity.
```

Comment:
Let's rewrite this as the following so we don't have the extra **s** outside the code block.
```
The identity of a [`Selector`].
```